### DISCUSSION TITLE
利用光頭的反射光可以讓孩子讀書嗎？

### DISCUSSION THREAD

**Discussion by dayu1004** [1101-GS4719-C/Python-Community-GS4719-C#524] [General]
偶然看到這個影片，這個企劃真的超有趣的。想說來分享這部影片也分享我的看法。

以下是影片網址：
https://youtu.be/zFuEi2aNkm8

首先是我發現節目組在架場的時候，相關位置上的一個小遺憾。
影片中，企劃說要利用光頭將光反射到孩子的小空間，希望可以讓他唸出紙上的靜思語；但我覺得小孩應該是要背對光頭們的，這樣才可以讓光線走 燈-光頭-紙張-孩子的眼睛 他會更容易看清楚紙上的內容。

再來是節目說想要參考LED排列的方式將光頭排成密集陣列，讓反射光的效果增加。雖然我跟LED不熟，但我覺得LED是發光體（也可以代表光源的一種），而光頭是反射面。在兩樣東西的本質上就不太一樣了，所以從同樣位置發出的光強也會有相當差距。

然後我又想了一下，影片裡的光源可以近似成一個有範圍的廣域平行光源，而光頭的頭假設成完美的光滑球面好了，從高中物理所學裡面可以知道凸面鏡和凹透鏡著一樣的效果。也就是說，多個光頭（多個凸面鏡）可以看成一個大的凸面鏡，因此它會將平行光源發散掉，使得孩子閱讀的那個區域光強會降低。但如果光源的光強度夠的話，搞不好真的可以讓孩子利用反射光讀書（只是在這之前，光頭的頭皮會先被燙傷吧🥲）。

最後，我覺得這個實驗除了光學之外，好像還有很多面向可以探討。造成影片裡實驗結果的原因有很多，綜合起來才會導致實驗沒有成功。但如果把字換成藍色或綠色的話，搞不好在全暗的狀態下人眼啟動暗視覺就可以真的看到一點點字⋯（好了！給我停下來🤣

最後的最後，不知道大家看完影片之後會有什麼想法呢？都歡迎在底下討論或分享對這部影片的心得。我自己看完之後真的是笑爆，這種看起來好荒謬但好像又可行的題材真的太有趣了！

**Comment by hapiwangy** [comment]
這種理論上合理但聽起來令人發笑的東西一直是可以吸引別人的東西。就類似搞笑諾貝爾獎帶給我們的感覺一樣。

**Comment by Yuhsin000** [comment]
光頭上如果有一些不平整處就會造成散射，不好抓他的效果ㄟ，感覺是搏君一笑的實驗，用光頭來製造話題性~

**Comment by xkllkx** [comment]
有一點日本早期綜藝節目的fu

**Comment by HuanYuuuuuu** [comment]
就當作搞笑影片看看就好，但的確蠻有趣的

**Comment by stan890425** [comment]
要在多上點蠟才會亮XD

**Comment by jcpan821** [comment]
小心!!看太久傷眼睛

**Comment by minniewei** [comment]
我覺得這是一個有趣的發現!!!

**Comment by yenwei12138** [comment]
這種真的很像日本綜藝節目ㄟ，一本正經搞笑的題材

**Comment by Ben6uo** [comment]
會不會引很多蟲蟲來倍讀?

**Comment by etin2468** [comment]
這光頭反射率...夠嗎?而且還是要有光源阿?

**Comment by Lynn12378** [comment]
原本已經禿頭了，再被強光照的話感覺真的再也長不出頭髮了XD

**Comment by Jimmy911218** [comment]
好有趣喔~謝謝樓主分享

**Comment by Hchejxnc** [comment]
企劃的確滿有趣的~

**Comment by ruthchen1020** [comment]
這是在測試光頭的反射光的程度嗎...

**Comment by StevenWanglolz** [comment]
這真的好像緯來日本台會播的實驗節目，好有趣~

**Comment by wanderer657** [comment]
這部影片展現了光學實驗的趣味性和效果，但也凸顯了一些改進點。例如，孩子站的位置可能不夠理想，導致反射光無法充分照射目標物。提及將光頭排列成LED，實際效果可能有所差異。這些觀點反映了對實驗設計的挑戰，呼籲找到更實用的解決方案。因此，通過這樣的討論和探索，我們能夠更好地理解光學實驗的複雜性，並提出更具創新性的方法。
